source/about/faq-integrations.rst

```diff
@@ -8,4 +8,54 @@ Mattermost is an open source, self-hosted alternative to proprietary SaaS servic
 
 While you're welcome to use the Mattermost source code under its open source license, Mattermost, Inc. does not offer support or technical advice for proprietary SaaS projects that result in customers potentially being paywalled from their data should they stop paying SaaS fees.
 
-To learn more about why we strongly believe that users and customers should always have access to their data, please read `why we created Mattermost <https://mattermost.com/about-us/>`__.
\ No newline at end of file
+To learn more about why we strongly believe that users and customers should always have access to their data, please read `why we created Mattermost <https://mattermost.com/about-us/>`__.
+
+What's the difference between incoming and outgoing webhooks?
+-------------------------------------------------------------
+
+A webhook is a way for one app to send real-time data to another app.
+
+In Mattermost, incoming webhooks receive data from external applications and make a post in a specified channel. They're great for setting up notifications when something happens in an external application.
+
+Outgoing webhooks take data from Mattermost and send it to an external application. The receiving server of the outgoing webhook can then post a response back in Mattermost. They're great for listening for a specific word, and then notifying external applications when a trigger word is used.
+
+What's a slash command?
+-----------------------
+
+A slash command is similar to an outgoing webhook, but instead of listening to a channel it is used as a command tool. This means if you type in a slash command it will not be posted to a channel, whereas an outgoing webhook is only triggered by posted messages.
+
+What does Slack-compatible mean?
+--------------------------------
+
+Slack compatible means that Mattermost accepts integrations that have a payload in the same format as Slack. If you have a Slack integration, you should be able to set it up in Mattermost without changing the format.   
+
+What if I have a webhook from somewhere other than Slack?
+---------------------------------------------------------
+
+If you have an integration that outputs a payload in a different format, you need to write an intermediate application to act as a translation layer to change it to the format Mattermost uses. Since there’s currently no general standard for webhook formatting, this is unavoidable and just a part of how webhooks work.
```

Comment:
```suggestion
If you have an integration that outputs a payload in a different format, you need to write an intermediate application, such as N8N.io, Zapier, or Integromat, to act as a translation layer to change it to the format Mattermost uses. Since there’s currently no general standard for webhook formatting, this is unavoidable and just a part of how webhooks work.
```